app/js/components/whatShouldIFix/whatShouldIFix.directive.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const componentsModule = require('../');
+const orderBy = require('lodash/orderBy');
+const map = require('lodash/map');
+
+/**
+ * @ngInject
+ */
+function whatShouldIFixCtrl($scope, Rule, Severities) {
+    Rule.getRulesLatest()
```

Comment:
The API should be updated to support these types of queries natively (instead of the frontend fetching all of the rules and then sorting).